hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1036,11 +1035,19 @@ func (r *HostedClusterReconciler) reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Set HostedCluster payload type
+	payloadType, err := r.determineHostedClusterPayloadType(ctx, hcluster, registryClientImageMetadataProvider)
+	if err != nil {
+		return ctrl.Result{}, err
+	}
+
+	log.Info("HostedCluster payload type set to: " + string(payloadType))
```

Comment:
This would be a verbosity logging every single reconcile, can we drop it?

Reply:
Removed